frog/imports/client/App/index.js

```diff
@@ -401,12 +401,14 @@ export default class Root extends React.Component<
       return (
         <ErrorBoundary>
           {!this.state.connected && <ConnectionDiv />}
-          <Router>
-            <Switch>
-              <Route path="/:slug" component={FROGRouter} />
-              <Route component={FROGRouter} />
-            </Switch>
-          </Router>
+          <ModalController>
```

Comment:
NIT: Maybe add a small comment explaining you `<ModalController>` is wrapped here.

Reply:
Addressed